README.md

```diff
@@ -33,11 +33,10 @@
 <p align="center" style="color:grey;"><i>Get started with Kestra in 4 minutes.</i></p>
 
 
-# Kestra Plugin Template
+# Kestra DLT Pugin
 
-> A template for creating Kestra plugins
+> A DLT plugin for Kestra
```

Comment:
Could you add more details and add a link to [the tool website](https://dlthub.com/)? So that we would have more context